python/setup.py

```diff
@@ -42,7 +42,7 @@
 for lang in langs:
     lang_titlecase = lang.title()
     # Only classifiers listed in https://pypi.org/classifiers/ are allowed
-    if lang_titlecase not in ('Armenian', 'Yiddish'):
+    if lang_titlecase not in ('Armenian', 'Estonian', 'Yiddish'):
         classifiers.append('Natural Language :: ' + lang_titlecase)
```

Comment:
```suggestion
    # Remove them here or submit them to https://github.com/pypa/trove-classifiers
```

Latest trove-classifiers now has these three languages.